src/codec/framed_read.rs

```diff
@@ -74,7 +74,7 @@ impl<T> FramedRead<T> {
 
         let kind = head.kind();
 
-        log::trace!("    -> kind={:?}", kind);
+        tracing::trace!("    -> kind={:?}", kind);
```

Comment:
stuff like this seems like it would be pretty trivial to turn into a structured field...but we can do that in a follow up